whitepaper/whitepaper.md

```diff
@@ -518,51 +518,53 @@ gets replaced by a traditional bank account holding USD.
-| Failure Criteria           | 1. It was not migrate the assets. <br> It was not possible to reconstruct the asset history on the target blockchain.                                                                                                                                                                                                                                                                                                                                                                                                          |
-| Prerequisites              | 1. All members belonging to `Consortium A` want to migrate the blockchain. <br> 2. The `Consortium A` controls the source blockchain. <br>3. `Consortium A` has write and execute permissions on the target blockchain<br>
-| Comments                   | An asset is defined as data or smart contracts originating from the source blockchain. <br> This use case relates to use cases implying asset portability (e.g., 2.1) <br> This use case provides blockchain portability, thus reducing costs and fostering blockchain adoption.                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
-
+| Use Case Attribute Name    | Use Case Attribute Value |
+| -------------------------- | -------------------------------------------- |
+| Use Case Title             | Blockchain Migration |
+| Use Case                   | 1. `User A` runs an application/service on a source blockchain that is either permissioned or permissionless.<br>2. `User A` decides to use another blockchain instance (permissioned or permissionless) to run his/her application. <br>3. `User A` migrates the existing assets and data to the target blockchain.<br>4. `User A` runs the application/service on the target blockchain.
+| Interworking patterns      | Value transfer, Data transfer |
+| Type of Social Interaction | Asset and Data Transfer |
+| Narrative                  | `User A` who's application/service is operating on a source blockchain (e.g., a Hyperledger Fabric instance), would want to migrate the assets, data, and functionality to a target blockchain (e.g., Hyperledger Besu) to expand their reach. However, such migration requires lots of technical effort and time. The `Blockchain Migration` feature from `Cactus` can provide support for doing so, by connecting to the source and target blockchains, and performing the migration task.
```

Comment:
I saw that proposed texts eliminated the description about therole of "Consortium A" and "Consortium B". I believe if the migration is intended to transfer the ownership or origination, both consortium should ask participant to authorize the transfer during preparation phase.
I also want to see text about how legacy data is treated on "targeted blockchain". 
I think it should have some reference to the transactions on the source blockchain and targeted blockchain continue to add new records for the future.

Reply:
Could you please explain a bit about 1st comment. I'm not clear about "... transfer the ownership or origination". Also, by "preparation phase" do you mean the "Initialization phase" in sequence diagram?

Legacy data can be transferred to target blockchain under several data fidelity levels ranging from no transfer to entire history depending on the use case. As per our work, we found 5 data fidelity levels, namely Fresh Start (nothing transfer), State Only, State and Transactions, Genesis and Transactions, Blockchain History (everything transfer). Also, we found 5 cases of migration (Relocate, Upgrade, Consolidate, Separation, and Archive). So whether there's some reference to transactions on source blockchain depends on the application. Should I add such details?

Reply:
I thought the assets recorded on source blockchain will be invalidated under agreement between participants on both source and targeted blockchain.
I was not convinced on the simple operation can do it.
There are two problem here,: 
1) how the participants of targeted ledger can trust the imported records from source ledger, 
2) how the transferred asset, which should not be exist, can be realized as "invalid" by the participants on sourced ledger without knowing/referencing targeted ledger.  
I suggested you to add some procedure for solving above issues.

Reply:
1. Consortium users have to trust some intermediatory (gateway, relay, inter-blockchain connector, or oracle) that shows the source blockchain data/assets to the target, e.g., proof of burn. This is where Cactus can help. This is similar to use case 3.6 where ledger 1 gets what it's missing from leader 2 (vice versa) through Cactus.
2. Assets can be burned and smart contracts can be destroyed (both typically refer as burning).
Expanded Narrative to include following:
"Assets on the source blockchain are burned and smart contracts are destructed during the migration to prevent double-spending. Proof-of-burn is verified on the target blockchain before creating the assets, smart contracts, or their state. ... 
As assets are burned and smart contracts are destructed, any future attempt to manipulate them on the source blockchain will fail. ... 
The `Blockchain Migration` feature from `Cactus` can provide support for doing so by connecting source and target blockchains; proving values and proof-of-burn of assets, smart contracts, and data imported from the source blockchain to the target; and then performing the migration task."

Reply:
I believe 'some intermediatory' should be authorized by the consortium member with obvious consensus or agreement which might be outside of the system. Can you add some text to clarify this fact?

For your comment-2,  that can be happened only when all members agreed on burning/creating asset across source/destination leader with steps. 
1-user request smart-contract on destination ledger (via CACTUS) to transfer his/her asset, but it will be pending until confirmation or authorization is given from smart-contract on sourced ledger.
2-user request smart-contract on sourced ledger (via CACTUS) to burn his/her asset
3- smart-contract will be notified confirmation or authorization from sourced ledger smart-contract 
4- smart-contract on destination ledger create his/her asset, as the response to confirmation/authorization on step3)

I would like to ask you to rewrite the section to capture the above observation.

Reply:
1. Add a sentence on use of intermediatory.
2. Added 4 steps suggested